.circleci/config.yml

```diff
@@ -19,7 +19,7 @@ jobs:
             - v2-deps-cache-{{ .Branch }}-{{ checksum "mix.lock" }}
             - v2-deps-cache-{{ .Branch }}
             - v2-deps-cache
-      - run: mix deps.get
+      - run: mix do deps.clean --all, deps.get, deps.compile
```

Comment:
this doesn't make much sense I'm afraid

* `deps.clean --all` is going to *empty* the `deps` directory (the cache we've just restored in the line above)
* `deps.compile` is cool, but doesn't need to happen at this point. Dependencies are compiled as part of `mix compile` and cached in the "build cache"

please revert this

Reply:
Hi @maltoe! The CI failed because of this.
Above the CI error:
```
#!/bin/bash -eo pipefail
mix test
** (Mix) Could not start application ranch: exited in: :ranch_app.start(:normal, [])
    ** (EXIT) an exception was raised:
        ** (UndefinedFunctionError) function :ranch_app.start/2 is undefined (module :ranch_app is not available)
            (ranch 1.8.0) :ranch_app.start(:normal, [])
            (kernel 7.1) application_master.erl:277: :application_master.start_it_old/4

Exited with code exit status 1
CircleCI received exit code 1
```
Using `- run: mix do deps.clean --all, deps.get, deps.compile` the error is gone.

Reply:
I think the cache works anyways.
The first workflow takes 6m58s and the rest a max of 2m30s.
https://app.circleci.com/pipelines/github/williamthome/chromic_pdf?filter=all

Reply:
You should have the same effect, by simply changing the cache version from v2 to v3 (which is usually how to clear the cache on circleci) 
I guess the cache has been build with an older erlang version I think, therefore we end up with this error every time. 
Your code will still use the cache for downloading, but not for compilation, which is still a huge portion of the time.

Reply:
> I think the cache works anyways. The first workflow takes 6m58s and the rest a max of 2m30s.

The speed-up you've seen is likely due to the other caches (compile cache, dialyzer, etc.), because each of these will have been rebuilt with the updated Erlang version and hence are now faster again. The deps cache is definitely useless if we delete the deps right after restoring them from cache ;)

Reply:
> changing the cache version from v2 to v3

👍  Though I guess now that we have rebuilt everything already we don't have to bump the cache number anymore. Just removing this change should be fine.

Reply:
Ok. 
I reverted the change at 4aa7afaa05ddd3ad2839ac30c00fec59bab42114.
Should mark this as resolved? Or how I can resolve this issue?